app/controllers/users_controller.rb

```diff
@@ -1,24 +1,25 @@
 class UsersController < ApplicationController
   before_action :require_login, only: %w[edit update destroy]
   before_action :set_user, only: %w[show]
-  http_basic_authenticate_with name: 'intern', password: ENV['INTERN_PASSWORD'] || 'test'
+  http_basic_authenticate_with name: 'intern', password: ENV['INTERN_PASSWORD'] || 'test', only: %i(new create)
```

Comment:
Line is too long. [111/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.